README.md

```diff
@@ -182,6 +182,102 @@ You can intermittently poll whether the operation has finished by using the `Pol
   }
 ```
 
+#### `Azure::Core::Context`
+
+Most Azure SDK Service Client methods accept an optional `Azure::Core::Context` parameter, which is used to enable cancellation of the operation or to
```

Comment:
Most.

Specifically, all "service methods" accept a `Context` parameter. But not all client methods are service methods.

As a degenerate example, the constructor does not take a Context parameter, because the constructor is not a service method.

But while the Azure SDK team has a distinguished category of APIs known as "service methods", we don't typically expose it as a discrete category of APIs to customers.